content/blog/wingingit/winging-it-24.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Mixins & Functions to Streamline CSS
+episode: 24
+tags:
+  - CSS
+  - CSS Mixins & Functions
```

Comment:
@SondraE @mirisuzanne I know we have a handful of posts with this tag, but would it make more sense in the future to have these as separate tags?

Reply:
I don't know, maybe just because one ships before the other? But is it an issue to keep them together until that's useful?

Reply:
@mirisuzanne I think it is OK to leave these as is for now but I was thinking we may write posts in the future that only has one or the other in the code snippets, or announcing when one rolls out, for instance. We don't have to do anything at the moment.